tests/main/install-hook/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Check install and remove hooks.
+
+prepare: |
+    echo "Build basic test package"
+    snapbuild $TESTSLIB/snaps/snap-install-hooks .
+    snapbuild $TESTSLIB/snaps/snap-install-hooks-broken1 .
+  
+restore: |
+    rm snap-install-hooks_1.0_all.snap
+    rm snap-install-hooks-broken1_1.0_all.snap
+
+execute: |
+    echo "Installing a snap with hooks"
+    snap install --dangerous snap-install-hooks_1.0_all.snap
+
+    echo "Verify configuration value with snap get"
+    snap get snap-install-hooks installed | MATCH 1
+    snap get snap-install-hooks foo | MATCH bar
+
+    echo "Verify that install hook is run only once"
+    snap set snap-install-hooks installed=2
+    snap install --dangerous snap-install-hooks_1.0_all.snap
+    snap get snap-install-hooks installed | MATCH 2
+
+    REMOVE_HOOK_FILE=/root/snap/snap-install-hooks/current/remove-hook-executed
+
+    echo "Verify that remove hook is not executed when removing single revision"
+    snap remove --revision=x1 snap-install-hooks
+    if test -f $REMOVE_HOOK_FILE
+    then
+      echo "Remove hook was executed. It shouldn't."
```

Comment:
Indentation is broken (we are using 4 spaces), in each of the remainder `if` blocks too.